### PR TITLE
Added the ability to edit and save reminders (issue 5)

### PR DESCRIPTION
**Pull request by sikemausa**
Purpose

Add the ability to edit and save a reminder

Approach

Allow the editing of reminders through "isEditing" property

Learning

Consulted the ember docs and discussed with classmates

https://guides.emberjs.com/v2.0.0/models/creating-updating-and-deleting-records/

Blog Posts

Open Questions and Pre-Merge TODOs

 Use Github checklists. When solved, check the box and explain the answer.
Test coverage

Tested to verify that the reminder was updated with the value of the input field after clicking "save"

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const Router = Ember.Router.extend({
 Router.map(function() {
   this.route('reminders', { path: '/' }, function() {
     this.route('reminder', { path: '/:id' });
+    this.route('new');
```
Comment (brittanystoroz) [inline app/router.js]: 💯

File: app/routes/reminders/new.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+});
```
Comment (brittanystoroz) [inline app/routes/reminders/new.js]: Is this file doing anything/needed for something to work?

File: app/templates/reminders.hbs [hunk 3]
```diff
@@ -1,14 +1,19 @@
-<div class="reminders-template">
   <h2>Reminders: </h2>
 
   <div class="reminders">
     {{#each model as |reminder|}}
       {{#link-to "reminders.reminder" reminder.id class="reminder-title"}}
-        <p class="spec-reminder-item">{{reminder.title}}</p>
+      <p class='spec-reminder-item'>
```
Comment (brittanystoroz) [inline app/templates/reminders.hbs]: Are we using single quotes everywhere or double quotes?? Consistency is key :) I know it's hard to remember when you don't have a linter/beautifier yelling at you but it's good to get in the habit of picking one or the other.

File: app/routes/reminders/new.js (hunk 2 above)
Comment (brittanystoroz) [inline app/routes/reminders/new.js]: Another strategy would be to delete it and see if it breaks anything ;)  From my perspective, it *looks* like you shouldn't need it, but I've been wrong in the past. So give it a try and report back.

File: app/templates/application.hbs [hunk 4]
```diff
@@ -1,4 +1,4 @@
 <div class="application template">
   <h1>Welcome to Remember</h1>
-  {{outlet}}
 </div>
+{{outlet}}
```
Comment (brittanystoroz) [inline app/templates/application.hbs]: Any reason in particular we're moving this outlet outside the container div?

File: app/templates/reminders/new.hbs [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<h2>New Reminder</h2>
+<form class="add-reminder-form" {{action "addReminder" on="submit"}}>
+ <label>
+   Title
+   {{input class="spec-input-title" value=title}}
```
Comment (brittanystoroz) [inline app/templates/reminders/new.hbs]: Might want a type attribute for this input just to be exceptionally clear.

File: app/templates/application.hbs (hunk 4 above)
Comment (bcgodfrey91) [inline app/templates/application.hbs]: I went ahead and did that on every file. When I had them inside, the app wasn't rendering properly, but when I moved them outside it did. Is it typically convention to place them inside?

Comment (brittanystoroz) [inline app/templates/application.hbs]: I'll have to look into it. I wasn't sure, which is why I asked. I'll take your word for it though.

File: app/routes/reminders/new.js (hunk 2 above)
Comment (bcgodfrey91) [inline app/routes/reminders/new.js]: Looks like there's no new issues. Good call!

**Pull request merged**